src/components/20-molecules/policy-features/policy-features-item/ui.test.js

```diff
@@ -0,0 +1,115 @@
+import { Selector } from 'testcafe';
+
+const host = process.env.TEST_HOST_STORYBOOK_URL || 'http://localhost:9999';
+const TAG = 'axa-policy-features-item';
+const CLASS = '.policy-features-item';
+const defaultWindowHeight = 1000;
+const lgWindowWidth = 992;
+const mdWindowWidth = 768;
+const smWindowWidth = 576;
+
+// Selectors
+const firstSvgElementInDocument = Selector(
+  () => document.querySelector('axa-policy-features-item').shadowRoot
+).find('svg');
+const firstH1ElementInDocument = Selector(
+  () => document.querySelector('axa-policy-features-item').shadowRoot
+).find('h1');
+const firstPElementInDocument = Selector(
+  () => document.querySelector('axa-policy-features-item').shadowRoot
+).find('p');
+const firstArticleElementInDocument = Selector(
+  () => document.querySelector('axa-policy-features-item').shadowRoot
+).find('article');
+
+fixture('Policy features item - basic functionality').page(
+  `${host}/iframe.html?id=molecules-policy-features--policy-features`
+);
+
+test('should render policy-features-item', async t => {
+  const $axaElem = await Selector(TAG);
+  await t.expect($axaElem.exists).ok();
+  const $axaElemShadow = await Selector(
+    () => document.querySelector('axa-policy-features-item').shadowRoot
+  );
+  const $axaElemShadowEl = await $axaElemShadow.find(CLASS);
+  await t.expect($axaElemShadowEl.exists).ok();
+});
+
+test('should render title of first item', async t => {
+  await t.expect(firstH1ElementInDocument.textContent).eql('Get Discount');
+});
+
+test('should render description of first item', async t => {
+  await t
+    .expect(firstPElementInDocument.textContent)
+    .eql('A 5 star car insurance with affordable premium services');
+});
+
+test('should render svg icon of first item', async t => {
+  await t.expect(firstSvgElementInDocument.exists).ok();
+});
+
+test('should set svg size for screen size md-up', async t => {
+  await t
+    .expect(await firstSvgElementInDocument.getStyleProperty('width'))
+    .eql('96px')
+    .expect(await firstSvgElementInDocument.getStyleProperty('height'))
+    .eql('96px');
+}).before(async t => {
+  await t.resizeWindow(mdWindowWidth, defaultWindowHeight);
+});
+
+test('should set svg size for screen size smaller than md', async t => {
+  await t
+    .expect(await firstSvgElementInDocument.getStyleProperty('width'))
+    .eql('42px')
+    .expect(await firstSvgElementInDocument.getStyleProperty('height'))
+    .eql('42px');
+}).before(async t => {
+  await t.resizeWindow(smWindowWidth, defaultWindowHeight);
+});
+
+test('should render title and description with size medium-1 on screen md', async t => {
```

Comment:
I would call this `should render title and description with correct typography on tablet view`. This makes it more abstract and if anyone decides that medium-1 will be renamed at some point in time, it won't diverge with this description.

Reply:
Yes, "correct size" is better than "medium-1". Will change that. "Tablet view" is readable very good, but do we have a specification for that? Or what is "sm"? A tablet or a big smartphone?